news/2023/2023-03-02-osumonthly-8.md

```diff
@@ -0,0 +1,257 @@
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/Mq3PfNScjWA" frameborder="0" allowfullscreen></iframe>
+
+## Skinning recap
+
+### Skinning news
+
+#### Website update makes searching for newest skins easier
+
+The osu! website has received an update, more specifically now you can sort threads on the forums by their creation date, which should hopefully make your skin browsing experience much easier.
+
+![](/wiki/shared/news/2023-03-02-osumonthly-8/forum-creation-date-sort.png)
+
+### Featured skins of the month
+
+You know the drill by now. 3 skins hand-picked by the skinship team that were made in the past month will now be presented in glorious fashion for your eyes to feast on.
+
+- [M I K A](https://osu.ppy.sh/community/forums/topics/1726217?n=1) - Staying on the topic of skins centred around characters from games, this one is made with Mika from *Blue Archive* being a centrepiece. Anime girls aside, the skin is heavily dominated by soft, pastel-like colours across all menus which are relaxing to look at.
+- [ArtemFauna](https://osu.ppy.sh/community/forums/topics/1716182?n=1) - This skin is based on VTuber [Ceres Fauna](https://www.youtube.com/@CeresFauna), and is in line with the theme of the character in question using green hues as the dominant colour, as well as the general nature theme being present both in gameplay and menus.
+- [kokopium](https://osu.ppy.sh/community/forums/topics/1728192?n=1) - Heavily themed around Kokomi from *Genshin Impact* (with a dash of Raiden as well), this skin takes various elements from the game and inserts them into osu!. If you are a fan of Kokomi, or just *Genshin Impact* in general, you will surely enjoy this skin.
+
+Check out all the skins highlighted with this showcase video below:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/videoseries?list=PLA2Nwtxa4WMpC1mSZ8oExgKkttfwvh_Mr" frameborder="0" allowfullscreen></iframe>
+
+## Did you know?
+
+In the earliest osu! days, a separate program was used to update osu! called "osume". Besides its function of updating the game, skins that were deemed finished and of good quality were also made downloadable straight through the updater.
+
+![](/wiki/shared/news/2023-03-02-osumonthly-8/did-you-know-feb23.png)
+
+## Monthly community quiz
+
+For the second month in a row, the quiz question for February has remained unanswered by anyone! 
+
+The correct answer was [Reol - +Danshi](https://osu.ppy.sh/beatmapsets/300748#osu/674514), a *loved* map where all of the map's object are placed in a pleasing circle shape. As for why it is referred to as the *circle of life*... maybe consider asking your parents.
+
+Now, on to this month's question:
+
+**What would the aristocracy have to do with osu!(lazer)?**
+
+chromb incorporated wishes everyone the best of luck in finding the answer to this THRILLING question and remember — there's a month of osu! supporter up for grabs for the first person to guess the correct answer!
```

Comment:
was about to reword to "chromb incorporated some wishes to everyone for the best of luck..."

```suggestion
chromb incorporated™ wishes everyone the best of luck in finding the answer to this THRILLING question and remember — there's a month of osu! supporter up for grabs for the first person to guess the correct answer!
```